CHANGELOG.md

```diff
@@ -274,7 +277,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Version 1.21.0/0.42b0 (2023-11-01)
 
 - Fix `SumAggregation`
-￼  ([#3390](https://github.com/open-telemetry/opentelemetry-python/pull/3390))
+  ￼ ([#3390](https://github.com/open-telemetry/opentelemetry-python/pull/3390))
```

Comment:
Here there is some strange unicode chars instead of a space, the parens should be aligned to the `F`

Reply:
Sorry, figured out it was my ide setting.